shared/query_engine/query_creator.py

```diff
@@ -25,13 +25,6 @@ def __init__(self, session, project, member, directory = None):
         self.session = session
         self.member = member
         self.directory = directory
-        # Additional security check just for sanity
-        Project_permissions.by_project_core(
```

Comment:
This is a duplicate. 
Context is that this is called later in SqlAlchemyQueryExecutor

Part of larger goal of aligning permissions into the same place

Reply:
Also that in general we are trying to put more validation at execution stage rather then prep/build stage (e.g dataset IDs which rely on the query string builder to format it first)